File: src/API_запрос.java
public class API_запрос {
    String querty;
    String token;

    public API_запрос(String тип, String токен) {
        querty = тип;
        token = токен;
    }

    public String toHTTPs() {
        return "https://www.myarena.ru/api.php?query=" + querty + "&token=" + token;
    }

    @Override
    public String toString() {
        return toHTTPs();
    }
}

File: src/API_запрос.kt
open class API_запрос(internal var querty: String, internal var token: String) {

    open fun toHTTPs(): String {
        return "https://www.myarena.ru/api.php?query=$querty&token=$token"
    }

    override fun toString(): String {
        return toHTTPs()
    }
}
